Windows gstreamer CI install failures

Things getting even stranger:
```
==> Downloading GStreamer 1.22.12...
Invoke-WebRequest: D:\a\qgroundcontrol\qgroundcontrol\tools\setup\install-dependencies-windows.ps1:119
Line |
 119 |      Invoke-WebRequest "$gstBaseUrl/gstreamer-1.0-msvc-x86_64-$gstVers …
     |      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     |                Checking you are not a bot                                                                     
     | .dropdown:hover > .menu { display: block; }             .ui.secondary.menu .dropdown.item > .menu { margin-top:
     | 0; }                                                                                                            
     | Checking you are not a bot                                                                Loading challenge
     | meta-refresh...                                                                    Why am I seeing this?        
     | You are seeing this because the administrator of this website has set up go-away to protect the server against
     | the scourge of AI companies aggressively scraping websites.   Mass scraping can and does cause downtime for the
     | websites, which makes their resources inaccessible for everyone.   Please note that some challenges requires the
     | use of modern JavaScript features and some plugins may disable these. Disable such plugins for this domain (for
     | example, JShelter) if you encounter any issues.                                                                 
     | Guru Meditation: b877a2e96413e07baead5ab54f847f01                                                               
     | Protected by go-away :: Request Id b877a2e96413e07baead5ab54f847f01
Error: Process completed with exit code 1.
```

--- a/tools/setup/install-dependencies-windows.ps1
+++ b/tools/setup/install-dependencies-windows.ps1
@@ -70,7 +70,8 @@ $installGst = $arch -eq 'AMD64'
 if ($installGst) {
     $gstInstallDir = 'C:\gstreamer'  # MSI INSTALLDIR (root)
     $gstPrefix     = 'C:\gstreamer\1.0\msvc_x86_64'  # Actual path after install
-    $gstBaseUrl    = "https://gstreamer.freedesktop.org/data/pkg/windows/$gstVersion/msvc"
+    # We now download from S3 because the official GStreamer site is unreliable
+    $gstBaseUrl    = "https://qgroundcontrol.s3.us-west-2.amazonaws.com/dependencies/gstreamer/windows/$gstVersion"
     $gstRuntime    = Join-Path $tempDir 'gstreamer-runtime.msi'
     $gstDevel      = Join-Path $tempDir 'gstreamer-devel.msi'
 }
@@ -115,7 +116,7 @@ function Add-ToPath {
 # 5) Download + install GStreamer
 # ────────────────────────────────
 if ($installGst) {
-    Write-Host "`n==> Downloading GStreamer $gstVersion..."
+    Write-Host "`n==> Downloading GStreamer version $gstVersion from $gstBaseUrl"
     Invoke-WebRequest "$gstBaseUrl/gstreamer-1.0-msvc-x86_64-$gstVersion.msi" -OutFile $gstRuntime
     Invoke-WebRequest "$gstBaseUrl/gstreamer-1.0-devel-msvc-x86_64-$gstVersion.msi" -OutFile $gstDevel
 

